template.rb

```diff
@@ -44,17 +44,15 @@ def render_file(filename)
 # Add standard gems
 # =================
 
-gem "hashdiff", [">= 1.0.0.beta1", "< 2.0.0"]
-
 gem_group :development, :test do
   gem "awesome_print"
   gem "console_ip_whitelist", github: "firstdraft/console_ip_whitelist"
   gem "dotenv-rails"
   gem "grade_runner", github: "firstdraft/grade_runner"
   gem "pry-rails"
-  gem "sqlite3", "~> 1.3.6"
+  gem "sqlite3", "~> 1.4.1"
   gem "table_print"
-  gem "web_git"
+  gem "web_git", github: "firstdraft/web_get"
```

Comment:
This is the aforementioned typo. Unless there's a sneaky repo named `web_get`

Reply:
Nice catch! I made this change last and didn't re-create a project to check.